README_process.md

```diff
@@ -0,0 +1,59 @@
+# The HZV Quarantine Process
```

Comment:
👍 💯 - This is an awesome description! 

💭- Maybe we should create a separat folder to store further readme's. 
* project
   *  Overall-readme
   * Exercise 1
      * Exercise-1 readme
      * ...

What is your opinion on this?

Reply:
Yes, I think this is a great idea.